packages/namespaces/src/isActionFromNamespace.test.js

```diff
@@ -1,7 +1,9 @@
+import { NAMESPACE_PREVENTED } from './constants';
 import isActionFromNamespace from './isActionFromNamespace';
 
 const fooAction = { meta: { namespace: 'foo' } };
 const barAction = { meta: { namespace: 'bar' } };
+const defaultNamespaceAction = { meta: { namespace: NAMESPACE_PREVENTED } };
```

Comment:
NAMESPACE_PREVENTED is not a "default namespace" it is more like a namespace with special treatment. rename this to `actionWithPreventedNamespace` and update test descriptions accordingly ("action had its namespace prevented")

Reply:
done